VRDR/IJEMortality.cs

```diff
@@ -81,6 +81,7 @@ public IJEMortality(string ije, bool validate = true)
                 // Grab the field value
                 string field = ije.Substring(info.Location - 1, info.Length);
                 // Set the value on this IJEMortality (and the embedded record)
+                // Console.Error.WriteLine(" field= " + field + "offset = " + info.Location);
```

Comment:
This PR seems to have a lot of commented out Console Writelines and other code, my suggestion would be remove all the commented out code.